src/components/Header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import main_img from '../.././assets/logo.svg';
+import up_arrow from '../.././assets/up-arrow.svg';
+import git_img from '../.././assets/Vector.svg';
+import './Header.css';
+
+function Header() {
+    return (
+        <div className="Container">
+            <div className="Container_1">
+                <img src={main_img} alt="Watcher-Logo" />
+                <p>
+                    Watcher
+                    <span className="borderBottom"></span>
+                    <img src={up_arrow} alt="" />
+                </p>
+            </div>
+
+            <button>
```

Comment:
change the button to link/ anchor tag.
And add href="https://github.com/Waishnav/Watcher"